CHANGELOG.md

```diff
@@ -1,5 +1,49 @@
 # CHANGELOG
 
+## v0.5.0
+
+### What's New
+
+**Manage EdgeX Foundry system in OpenYurt in a cloud-native, non-intrusive way**
+
+- yurt-edgex-manager
+  Yurt-edgex-manager enable OpenYurt to be able to manage the EdgeX lifecycle. Each EdgeX CR (Custom Resource) stands for an EdgeX instance.
+Users can deploy/update/delete EdgeX in OpenYurt cluster by operate the EdgeX CR directly. ([yurt-edgex-manager](https://github.com/openyurtio/yurt-edgex-manager), [@yixingjia](https://github.com/yixingjia), [@lwmqwer](https://github.com/lwmqwer))
+
+- yurt-device-controller
+  Yurt-device-controller introduces an approach leverages existing edge computing platforms, like EdgeX Foundry, and uses Kubernetes custom resources to manage edge devices.
+These CRDs and corresponding controllers allow users to manage edge devices in a declarative way, which provides users with a Kubernetes-native experience and reduces the complexity of managing,
```

Comment:
Yurt-device-controller aims to provider device management functionalities to OpenYurt cluster by integrating with edge computing/IOT platforms, like EdgeX  in a cloud native way. 

It will automatically synchronize the device status to device CR (custom resource) in the cloud and any update to the device will pass through to the edge side seamlessly. More information can be found at:

Reply:
updated